app/models/comment.rb

```diff
@@ -12,6 +12,7 @@ class Comment < ApplicationRecord
   COMMENTABLE_TYPES = %w[Article PodcastEpisode].freeze
   TITLE_DELETED = "[deleted]".freeze
   TITLE_HIDDEN = "[hidden by post author]".freeze
+  MAX_USER_MENTIONS = 6
```

Comment:
I specifically chose six here because that is the same number of mentions [we plan to show](https://github.com/forem/rfcs/pull/16/files#diff-b1fe05dec5655c4a78f12ac288b96dfcb82cd5877e10a16064ef520210a9a064R48) in our upcoming @-mention autocomplete feature. But if folks disagree with this, I set it up so that it's easy to change :)